packages/fluent/scss/split-button/_layout.scss

```diff
@@ -52,6 +35,40 @@
 
         .k-button-icon {
             min-width: 0;
+
+            &::after {
+                content:"";
+                width: var( --kendo-split-button-arrow-delimiter-line-size, #{$kendo-split-button-arrow-delimiter-line-size} );
+                height: var( --kendo-split-button-arrow-delimiter-size, #{$kendo-split-button-arrow-delimiter-size} );
+                position: absolute;
+                top: calc( ( 100% - #{$kendo-split-button-arrow-delimiter-size} ) / 2 );
+                background: var( --kendo-split-button-arrow-delimiter-bg, #{$kendo-split-button-arrow-delimiter-bg} );
+                z-index: 2;
+            }
+        }
+
+        &.k-button-sm {
```

Comment:
I would suggest creating a map for this and defining the sizes in the variables file (similar to the sizes in the button). In this way it'll be easier if we add different button sizes in the future.